_publications/2017_jpf_dr.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: publication
+# The quotes make the : possible, otherwise you can do it without quotes
+title: "Consistency-Aware Scheduling for Weakly Consistent Programs"
+# YYYY_CONFSHORTNAME_PAPERSHORTNAME
+key: 2017_jpf_dr
+
+# conference | journal | poster | workshop
+type: workshop
+redirect_from: /publications/2017_jpf_dr
+
+# The shortname is used for auto-generated titles
+shortname: JPF 2017
+# add a 2:1 aspect ratio (e.g., width: 400px, height: 200px) to the folder /assets/images/papers/
+image: 2017_jpf_dr.png
```

Comment:
The actual file names have upper case PNG. So change that in file names of images.
Also, your teaser images are not both 1600x800. Please fix that too.

Reply:
Done.